samples/istio/quickstart/bookinfo-gateway.yaml

```diff
@@ -0,0 +1,67 @@
+apiVersion: networking.istio.io/v1alpha3
+kind: Gateway
+metadata:
+  name: bookinfo-gateway
+spec:
+  selector:
+    istio: ingressgateway # use istio default controller
+  servers:
+  - port:
+      number: 80
+      name: http
+      protocol: HTTP
+    hosts:
+    - "bookinfo.example.com"
+---
+apiVersion: networking.istio.io/v1alpha3
+kind: VirtualService
+metadata:
+  name: bookinfo
+spec:
+  gateways:
+  - mesh
+  - bookinfo-gateway
+  hosts:
+  - productpage
+  - "bookinfo.example.com"
+  http:
+  - match:
+    - uri:
+        exact: /productpage
+    - uri:
+        prefix: /static
+    - uri:
+        exact: /login
+    - uri:
+        exact: /logout
+    - uri:
+        prefix: /api/v1/products
+    route:
+    - destination:
+        host: productpage
+        port:
+          number: 9080
+        subset: productpage-v1
+      weight: 100
+    - destination:
+        host: productpage
+        port:
+          number: 9080
+        subset: productpage-v3
+      weight: 0
+---
+apiVersion: networking.istio.io/v1alpha3
+kind: DestinationRule
```

Comment:
I see productpage-v1, and v3, but no v2. Is there a reason? Renaming v3 to v2 will make things more straightforward.

Reply:
@kalantar, @sriumcp: Below is a reminder of the behavior of the 3 versions, according to the images we have built:

productpage-v1:
* no errors
* normal latency behavior (no injected delay)
* comparatively low reward

productpage-v2: 
* no errors
* higher latency (some noticeable delay injected)
* reward much much higher than that of v1 and v3

productpage-v3: 
* no errors
* normal latency behavior
* reward higher than v1 but lower than v2

Reply:
I left the version names we used from our A/B/n  tutorial in iter8 1.0.0. But I agree this is probably a good change to make.